test/transform/forEachTest.js

```diff
@@ -48,19 +48,6 @@ describe('For loops to for-each', () => {
       );
     });
 
-    it('should preserve original const-kind of loop variable', () => {
-      expectTransform(
-        'for (let i=0; i < array.length; i++) {\n' +
-        '  const item = array[i];\n' +
-        '  console.log(item);\n' +
-        '}'
-      ).toReturn(
-        'array.forEach(item => {\n' +
-        '  console.log(item);\n' +
-        '});'
-      );
-    });
-
```

Comment:
It's somewhat confusing that in tests we have a mix of `const` and `let`. I think we should pick just one, like `let` and add a separate test to ensure that `const` also works.